src/test/java/com/google/cloud/tools/maven/cloudsdk/CloudSdkDownloaderTest.java

```diff
@@ -85,7 +85,7 @@ public void testDownloadCloudSdk_installAppEngine()
   public void testDownloadCloudSdk_ignoreAppEngineComponent()
       throws ManagedSdkVerificationException, ManagedSdkVersionMismatchException {
     when(managedCloudSdk.isInstalled()).thenReturn(true);
-    when(managedCloudSdk.hasComponent(SdkComponent.APP_ENGINE_JAVA)).thenReturn(false);
+    lenient().when(managedCloudSdk.hasComponent(SdkComponent.APP_ENGINE_JAVA)).thenReturn(false);
```

Comment:
What does this do?

Reply:
Mockito fails if there is any stubbing that's not being used. In this test, no one called `hasComponent(...)`, so Mockito wants us to remove this `when()`. To make Mockito tolerate this, we can use `Mockito.lenient()` instead of removing `when()`. Removing it works too, especially knowing that the default return value for `boolean` is false anyway.

Reply:
ah, I see, then maybe we can just remove this line.